client/asset/btc/btc.go

```diff
@@ -820,9 +839,9 @@ func (btc *ExchangeWallet) split(value uint64, lots uint64, outputs []*output, i
 	if baggage > excess {
 		btc.log.Debugf("Skipping split transaction because cost is greater than potential over-lock. "+
 			"%d > %d", baggage, excess)
-		btc.log.Infof("Funding %d %s order with coins %v worth %d",
-			value, btc.walletInfo.Units, coins, coinSum)
-		return coins, false, nil
+		btc.log.Infof("Funding %s %s order with coins %v worth %s",
+			sat(value), btc.symbol, coins, sat(coinSum))
+		return coins, false, nil // why fundingCoins is an input
```

Comment:
> // why fundingCoins is an input

Nuke it

Reply:
This comment was supposed to be declaratory actually.  On return with `err==nil`, the deferred function processes the `fundingCoins`.  The `fundingCoins` arg is super confusing though in this regard.  It was removable in dcr, but not btc because of this, and I really didn't want to refactor too much stuff.  I'll clarify this comment.